libvast/src/system/spawn_catalog.cpp

```diff
@@ -27,14 +27,6 @@ spawn_catalog(node_actor::stateful_pointer<node_state> self,
     = detached
         ? self->spawn<caf::detached>(catalog, accountant, args.dir / args.label)
         : self->spawn(catalog, accountant, args.dir / args.label);
-  self->request(handle, caf::infinite, atom::load_v)
-    .await([](atom::ok) {},
-           [](caf::error& err) {
-             VAST_WARN("catalog failed to load taxonomy "
-                       "definitions: {}",
-                       std::move(err));
-             // TODO: Shutdown when failing?
-           });
```

Comment:
I think it is better to keep this here.

Reply:
Why?

Reply:
Because why should the index tell the catalog to load taxonomies from disk? Either it should be done automatically or here.

Reply:
IMO: Yes, as it doesn't make sense to use.

Reply:
The loading logic was moved to the actor spawning function now. This is the right place.